201-virtual-wan-with-all-gateways/README.md

```diff
@@ -0,0 +1,81 @@
+# Azure Virtual WAN (vWAN) Multi-Hub Deployment
+
+![Azure Public Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/201-virtual-wan-with-all-gateways/PublicLastTestDate.svg)
+![Azure Public Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/201-virtual-wan-with-all-gateways/PublicDeployment.svg)
+
+![Azure US Gov Last Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/201-virtual-wan-with-all-gateways/FairfaxLastTestDate.svg)
+![Azure US Gov Last Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/201-virtual-wan-with-all-gateways/FairfaxDeployment.svg)
+
+![Best Practice Check](https://azurequickstartsservice.blob.core.windows.net/badges/201-virtual-wan-with-all-gateways/BestPracticeResult.svg)
+![Cred Scan Check](https://azurequickstartsservice.blob.core.windows.net/badges/201-virtual-wan-with-all-gateways/CredScanResult.svg)
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-virtual-wan-with-all-gateways%2Fazuredeploy.json)
+
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-virtual-wan-with-all-gateways%2Fazuredeploy.json)
+
+## Solution Overview
+
+This template creates a fully functional Azure Virtual WAN (vWAN) environment with the following resources:
+
+- Two distinct hubs in different regions
+- Four Azure Virtual Networks (VNET)
+- Two VNET connections for each vWAN hub
+- One Point-to-Site (P2S) VPN gateway in each hub
+- One Site-to-Site (S2S) VPN gateway in each hub
+- One Express Route gateway in each hub
+
+## Architecture
+
+vWAN resource deployed is of type "Standard" with default full mesh connectivity.
+The scenario implemented is exactly the one referenced in the Azure Virtual WAN documentation article below:
+
+[Azure vWAN Routing Scenario: any-to-Any](https://docs.microsoft.com/azure/virtual-wan/scenario-any-to-any)
+
+:::image type="content" source="images/azurevirtualwanarchitecturefigure1.jpg" alt-text="vWAN Architecture":::
```

Comment:
this is not valide MD

Reply:
Interesting.... used VSCODE well-known extension for MD and this is what has been generated. Will fix it, thanks.